kubetest/gke.go

```diff
@@ -330,11 +331,17 @@ func (g *gkeDeployer) Up() error {
 			}
 		}
 	}
-	// TODO(zmerlynn): The version should be plumbed through Extract
-	// or a separate flag rather than magic env variables.
-	if v := os.Getenv("CLUSTER_API_VERSION"); v != "" {
-		args = append(args, "--cluster-version="+v)
+
+	if *gkeReleaseChannel != "" {
```

Comment:
Would this work with the custom cluster creation command (`--gke-create-command`)?

Reply:
yes, https://github.com/kubernetes/test-infra/blob/88ef2b18615c596c9972a4d2a4e28408208d58cf/kubetest/gke.go#L168